src/login/logind-user.c

```diff
@@ -891,7 +893,17 @@ int config_parse_user_tasks_max(
         assert(rvalue);
         assert(data);
 
-        /* First, try to parse as percentage */
+        if (isempty(rvalue)) {
+                *m = manager->user_tasks_max;
```

Comment:
I don't think this can work. *m *is*  manager->user_tasks_max. Most likely this should be initialized the same as in logind.c. But duplicating the definition (`system_tasks_max_scale(33U, 100U)`) would be ugly. Maybe a static inline function in logind.h?